core-web/libs/sdk/angular/README.md

```diff
@@ -63,7 +369,6 @@ export const DOTCMS_CLIENT_TOKEN = new InjectionToken<DotCmsClient>('DOTCMS_CLIE
 
 export const appConfig: ApplicationConfig = {
     providers: [
-        provideRouter(routes),
```

Comment:
around here, we need to change the way we initialize the token. Now its like this

```
export const DOTCMS_CLIENT_TOKEN = new InjectionToken<ReturnType<typeof createDotCMSClient>>(
    'DOTCMS_CLIENT'
);
```

Reply:
Done!